src/ceph-disk/ceph_disk/main.py

```diff
@@ -1251,7 +1251,7 @@ def get_conf_with_default(cluster, variable):
     except subprocess.CalledProcessError as e:
         raise Error(
             'getting variable from configuration failed',
-            e,
+            e, e.output,
```

Comment:
This looks OK, but what if the `CalledProcessError` is not coming from the modified one from `_check_output` and it is really coming from `subprocess` ? You wouldn't get the `output` that gets slapped there. Maybe a `getattr(e, 'output', '')` ?

Reply:
@alfredodeza, `CalledProcessError` is raised by `subprocess.check_call()`, `subprocess.check_output()` and  `subprocess.CompletedProcess.check_returncode()`. it won't be raised by `subprocess.Popen` 's constructor or `subprocess.Popen.communicate()`. i double checked the document and the source of py2 and py3

Reply:
Man is this difficult to follow. You are right, the `command()` is calling `Popen`.